.travis.yml

```diff
@@ -5,7 +5,6 @@ language: python
 
 env:
   matrix:
-    - PYTHON_VERSION="2.6" NUMPY_VERSION="1.6"
```

Comment:
Hmmm the tests pass now although you have not regenerated the data. This is a bit worrisome.